coronado/healthmonitor.py

```diff
@@ -32,13 +48,39 @@ def check(klass) -> object:
         return TripleObject(json.loads(response.content))
 
 
-if '__main__' == __name__:
-    # TODO:  Turn this into a real object and define its unit tests and a runner.
+# --- functions ---
+
+def main(unitTest : bool = False) -> dict:
+    """
+    Check the triple API service health by reporting the API version, build ID,
+    and other information relevant to implementers.
+
+    This function is the main entry point for a script that will be installed to
+    `/usr/local/bin/triplchk` or equivalent depending on whether the Coronado
+    package was installed in the system wide Python configuraiton or in a
+    virtual environment.
+
+    See:  `man 5 coronado` for details.
+
+    Arguments
+    ---------
+        unitTest
+    Set it to True for running it in a unit test.
+
+    Returns
+    -------
+        dict
+    `None` if running standalone, or a dictionary with information useful to
+    API developers if running a unit test.
+    """
     _config = auth.loadConfig()
     _auth = Auth(_config['tokenURL'], clientID = _config['clientID'], clientSecret = _config['secret'], scope = Scopes.PUBLISHERS)
 
     HealthMonitor.initialize(_config['serviceURL'], SERVICE_PATH, _auth)
     check = HealthMonitor.check()
 
-    print('API version = %s, build = %s' % (check.apiVersion, check.build)) # TODO: fix the camelCase converter
+    print('triple API version = %s, build = %s' % (check.APIVersion, check.build))
```

Comment:
I prefer `f` strings for their legibility.

```suggestion
    print(f'triple API version = {check.APIVersion}, build = {check.build}')
```